HW3_IzzyGaw.R

```diff
@@ -0,0 +1,115 @@
+####EX. if (condition) {break}
+##########
+
+
+
+##########################
+#########MY WORK##########
+##########################
+
+mysqrt <- function( x=81, tol=.000001, myguess=27, verbose=FALSE){  #This sets up the function and its arguments. It establishes the x and myguess (which both are used to find the square root), the tolerance (which is the amount of deviation allowed from the actual answer), and verbose (which if turned off will not show all the output of the conditionals (default), but if TRUE will show). 
+
+	newguess <- myguess #renames myguess to newguess
+	newguess  #check
+	
+	if (x>0) { #this makes it so you cannot use negatives in the input, and if you do, you get a custom error message
+	approxy <- x/newguess #divides x by newguess and renames approxy
+	approxy #check
+	newguess <- (newguess+approxy)/2 #divides the sum of new guess and approxy by 2 and renaming to newguess.
+	newguess #check
+	error = newguess - approxy #finding the error 
+	error <- abs(error) #making sure the error is positive by using the absolute value function
+	error < tol #made the tolerance higher than the error
+	if (verbose==T) {print(error < tol)} #if the user wanted, this function would print if the error is larger or smaller than the tolerance.
+	if (verbose==T) {print(newguess)} #if the user wanted, this function would print the value of newguess.
+	if (verbose==T) {print(error)} #if the user wanted, this function would print the value of the error.
+
+	while ( error > tol ) { #this function says, if the error is larger than the tolerance, then ...
+	  approxy <- x/newguess  #divide x/newguess
+	  newguess <- (newguess+approxy)/2  #add newguess and approxy then divide by 2
+	  error = newguess - approxy #subtract approxy from newguess
+	  error <- abs(error)  #find the absolute value of error and rename to "error"
+	  if (verbose==T) {print(error < tol)} #if the user wanted, this function would print if the error is larger or smaller than the tolerance.
+	  if (verbose==T) {print(newguess)} #if the user wanted, this function would print the value of newguess.
+	  if (verbose==T) {print(error)} #if the user wanted, this function would print the value of the error.
+	}
+	return( newguess ) # 
+	}else print("BAD. DO NOT USE NEGATIVES") #this is the custom error message
+
+}
+
+
+#Testing the above function with various values.
+
+mysqrt(16) #does not show conditionals
+mysqrt(16, verbose=TRUE) #shows conditionals
+mysqrt(-16) #does not show conditionals
+mysqrt(49)
+mysqrt(-49)
+mysqrt(49, verbose=TRUE)
+mysqrt(361)
+mysqrt(-361)
+mysqrt(361, verbose=TRUE)
+mysqrt(10000)
+mysqrt(-10000)
+mysqrt(10000, verbose=TRUE)
+mysqrt(810000)
+mysqrt(-810000)
+mysqrt(810000, verbose=TRUE)
+
+
+###HW2 RENDER####
+setwd("C:/Users/izzyg/Documents/rbootclass/SquareRoots-repo")
+require(rmarkdown)
+Sys.setenv(RSTUDIO_PANDOC="C:/Program Files/RStudio/bin/pandoc")
+render("HW3_IzzyGaw.Rmd", output_format="html_document") # to html
```

Comment:
This will cause two errors!  You have two lines that refer to paths particular to your own computer.
You should not do this here anyway, if anything write a separate script to generate the markdown.